Coils-Rebalance

Dys-the-Fox opened the pull request:
Corruption Coils
> Set was entirely too strong and despite the heavy debuffs, the strength it provided was far too high and became the best-in-slot for almost every single elemental build. Especially powerful on the newly reworked Vengeance skill.

Demonic Chuckle
• +100% to Fire Skill damage [3 Items] -> +40% to Fire Skill Damage [3 Items]
• Drain Life -20 [2 Items] -> Removed
• Drain Life-15 [3 Items] -> Removed
• +200% Magic Find [2 Items] -> +50% Magic Find

Evil Humor
• +100% to Lightning Skill Damage [3 Items] -> +40% to Lightning Skill Damage [3 Items]
• Drain Life -20 [2 Items] -> Removed
• Drain Life -15 [3 Items] -> Removed
• +200% Magic Find [2 Items] -> +100% Gold Find [2 Items]

Temptation's Death
• +100% to Cold Skill Damage [3 Items] -> +40% to Cold Skill Damage [3 Items]
• Drain Life -20 [2 Items] -> Removed
• Drain Life -15 [3 Items] -> Removed
• +200% Magic Find [2 Items] -> 10% Chance to cast Level 20 Taunt When Struck [2 Items]

Set Bonuses
• Drain Life -15 [2 Items] -> Drain Life -50 [2 Items]
• +10 to All Skills -> +5 to All Skills [Full Set]
• +100% to Poison Skill Damage [Full Set] -> +40% to Poison Skill Damage [Full Set]
• Drain Life -45 [Full Set] -> All Resistances -50 [Full Set]
• -90% to Experience Gained [Full Set] -> -40% to Experience Gained [Full Set]
• +100% Magic Find [Full Set] -> Unchanged

The-Honorable commented:
a small part of me thinks the full set should have the penalty, this would also impact legacy sets too instead of just giving them the new 15% bonus ele on top of the old individual set item bonuses. becuase as i understand it, only full set bonus is re-checked on login and not setitem bonuses.

so individual set bonuses would be just MF and the full set would have both the penalty and the extra dmg
- (-50) all res
- +40% extra-elem
- +7 ask

Dys-the-Fox commented:
> a small part of me thinks the full set should have the penalty, this would also impact legacy sets too instead of just giving them the new 15% bonus ele on top of the old individual set item bonuses. becuase as i understand it, only full set bonus is re-checked on login and not setitem bonuses.
> 
> so individual set bonuses would be just MF and the full set would have both the penalty and the extra dmg
> 
> * (-50) all res
> * +40% extra-elem
> * +7 ask

Ah okay, I didn't know that set bonuses was rechecked upon login. That's a good suggestion, then, and can put that on the full set bonus instead.

Delegus commented:
This combination is going to look pretty bad in game. You have 65% ele damage spread across 4 individual locations + extra elem on the full set, so it's going to show all 4 strings on the full set because poison will be 65 and the rest 40. Need to just put it all in the 4 individual %'s

Personally +50 dmg/-50 res and +5 ASK instead of +7 just satisfies the mental gymnastics but feel free to ignore this. :D

Dys-the-Fox commented:
> This combination is going to look pretty bad in game. You have 65% ele damage spread across 4 individual locations + extra elem on the full set, so it's going to show all 4 strings on the full set because poison will be 65 and the rest 40. Need to just put it all in the 4 individual %'s
> 
> Personally +50 dmg/-50 res and +5 ASK instead of +7 just satisfies the mental gymnastics but feel free to ignore this. :D

I removed the elemental bonus from the individual set pieces and instead put the full bonus on the full set, and removed the poison bonus as well. I believe it should show up as +40% to all elemental damage. Do you mean from the pre-patch items?

Dys-the-Fox commented:
> You have 25% of cold fire and light on ammy ring and ring, 25 poison on set, extra-elem on set. ATM you are giving 65% total.

I sure did, I apparently edited my local version and not the branch version, lol, so it didn't update properly.
